src/spec-common/injectHeadless.ts

```diff
@@ -349,12 +370,12 @@ export async function runLifecycleHooks(params: ResolverParameters, lifecycleHoo
 
 	params.output.write('LifecycleCommandExecutionMap: ' + JSON.stringify(lifecycleHooksInstallMap, undefined, 4), LogLevel.Trace);
 
-	await runPostCreateCommand(params, lifecycleHooksInstallMap, containerProperties, 'onCreateCommand', remoteEnv, false);
```

Comment:
nit: I know you didn't introduce this name, but consider while you're here renaming `runPostCreateCommand` to something more descriptive `runLifecycleHook(...)`?

Reply:
Good idea. I am thinking to keep that rename for a later commit/pr, to make it easy to review the core of this PR